app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -0,0 +1,106 @@
+        # when the retryable_error_handler is called
+        form_job_status = Form526JobStatus.find_by(job_id:)
+        bgjob_errors = form_job_status.bgjob_errors || {}
+        new_error = {
+          "#{timestamp.to_i}": {
+            caller_method: __method__.to_s,
+            timestamp:,
+            form526_submission_id:,
+            supporting_evidence_attachment_guid:
+          }
+        }
+        form_job_status.update(
+          status: Form526JobStatus::STATUS[:exhausted],
+          bgjob_errors: bgjob_errors.merge(new_error)
+        )
+
+        Rails.logger.warn(
+          'Form526DocumentUploadFailureEmail retries exhausted',
+          { job_id:, timestamp:, form526_submission_id:, error_class:, supporting_evidence_attachment_guid: }
+        )
+
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.exhausted")
+      rescue => e
+        ::Rails.logger.error(
+          'Failure in Form526DocumentUploadFailureEmail#sidekiq_retries_exhausted',
+          {
+            job_id:,
+            submission_id: form526_submission_id,
+            supporting_evidence_attachment_guid:,
+            pre_exhaustion_failure: {
+              error_class:
+            }
+          }
+        )
+        raise e
+      end
+
+      def perform(form526_submission_id, supporting_evidence_attachment_guid)
+        @notify_client ||= VaNotify::Service.new(Settings.vanotify.services.benefits_disability.api_key)
+        send_notification_mailer(form526_submission_id, supporting_evidence_attachment_guid)
+
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.success")
+      rescue => e
+        retryable_error_handler(e)
+      end
+
+      private
+
+      def send_notification_mailer(form526_submission_id, supporting_evidence_attachment_guid)
+        form_attachment = SupportingEvidenceAttachment.find_by!(guid: supporting_evidence_attachment_guid)
+        submission = Form526Submission.find(form526_submission_id)
+
+        # We need to obscure the original filename since it may contain PII
+        obscured_filename = form_attachment.obscured_filename
+
+        email_address = submission.get_email_address
+        first_name = submission.get_first_name
+        date_submitted = submission.get_formatted_creation_time
+        @notify_client.send_email(
+          email_address:,
+          template_id: mailer_template_id,
+          personalisation: { first_name:, filename: obscured_filename, date_submitted: }
+        )
+
+        Rails.logger.info(
+          'Form526DocumentUploadFailureEmail notification dispatched',
+          {
+            obscured_filename:,
+            form526_submission_id: submission.id,
+            supporting_evidence_attachment_guid:,
+            timestamp: Time.now.utc
+          }
+        )
+      end
+
+      def mailer_template_id
+        Settings.vanotify.services
+                .benefits_disability.template_id.form526_document_upload_failure_notification_template_id
```

Comment:
You may be able to define `Settings.vanotify.services.benefits_disability` as a privatge method and append a chained template id or api key where needed.

Reply:
Yeah not a bad idea to clean up this line, but leaning towards being direct here so the full chain is visible in the same place